judo-ui-react/src/main/java/hu/blackbelt/judo/ui/generator/react/UiWidgetHelper.java

```diff
@@ -335,19 +335,26 @@ public static Boolean displayDropdownForButtonGroup(ButtonGroup actionGroup) {
     }
 
     public static String tableButtonVisibilityConditions(Button button, Table table, PageContainer container) {
+        String result = "";
+        if (button.getEnabledBy() != null) {
+            result += "(ownerData ? ownerData." + button.getEnabledBy().getName() + " : true) && ";
+        }
+        if (button.getHiddenBy() != null) {
+            result += "(ownerData ? !ownerData." + button.getHiddenBy().getName() + " : true) && ";
+        }
         if (button.getActionDefinition().getIsOpenCreateFormAction() && !table.isIsEager() && container.isView()) {
-            return "!editMode && (isFormUpdateable ? isFormUpdateable() : false)";
+            return result += "!editMode && (isFormUpdateable ? isFormUpdateable() : false)";
```

Comment:
I recommend adding a space at the end of the string to avoid any syntax errors (like you correctly did on lines 340-343)